tests/TypecheckSpec.hs

```diff
@@ -139,6 +146,9 @@ _inferAccounts = inferModule False "examples/accounts/accounts.repl" "accounts"
 _inferCP :: IO ([TopLevel Node],[Failure])
 _inferCP = inferModule False "examples/cp/cp.repl" "cp"
 
+_inferYield :: IO ([TopLevel Node], [Failure])
```

Comment:
do we need this? the idea with the others is they check big bad modules as a kind of "catch anything" in ghci while iterating. I don't mind leaving these things in so just askin

Reply:
I don't think we need it for now. It was just leftover from the typecheck debugging work. I'll remove it